django/core/management/commands/runserver.py

```diff
@@ -114,8 +118,9 @@ def inner_run(self, *args, **options):
         shutdown_message = options.get('shutdown_message', '')
         quit_command = 'CTRL-BREAK' if sys.platform == 'win32' else 'CONTROL-C'
 
-        self.stdout.write("Performing system checks...\n\n")
-        self.check(display_num_errors=True)
+        if not options['skip_checks']:
+            self.stdout.write("Performing system checks...\n\n")
+            self.check(display_num_errors=True)
```

Comment:
Should this also print something like "Skipping system checks..." when --skip-checks is true ?

Reply:
I don't think so because we don't have this behavior for other management commands which have `--skip-checks` option